core/pubnub_subscribe_v2.c

```diff
@@ -136,16 +133,16 @@ enum pubnub_res pbcc_parse_subscribe_v2_response(struct pbcc_context* p)
     if (jonmpOK == jpresult) {
         struct pbjson_elem titel;
```

Comment:
Nitpick: Should this variable be `title` ?

Reply:
Actually, no, it's a short-form of "_ti_me _t_oken _el_ement". It's a kind of an inside joke, also, there's a small town called "Titel" in Serbia. :)